src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -79,6 +79,7 @@ def fetch_notifications
                     else
                       NotificationsFinder.new(notifications_for_subscribed_user).for_notifiable_type(params[:type])
                     end
+    params[:page] = notifications.page(params[:page]).total_pages if notifications.page(params[:page]).out_of_range?
     params['show_all'] ? show_all(notifications) : notifications.page(params[:page])
```

Comment:
Just to be consistent with the changes in `_notifications_list.html.haml`:
```suggestion
    params[:show_all] ? show_all(notifications) : notifications.page(params[:page])
```

Reply:
:+1: Changed.